### PR TITLE
Clarify installation instructions (resolves #348)

### LINKED ISSUE #348: Installation instructions are misleading
https://github.com/BD2KGenomics/toil-scripts/tree/master/src/toil_scripts/rnaseq_cgl#installation

When using `virtualenv --system-site-packages` one should not `pip install toil`.

### PR DESCRIPTION
**Pull request by jvivian**

### CODE REVIEW

**Comment by jvivian** [comment]
@hannes-ucsc — Ready for review

File: src/toil_scripts/exome_variant_pipeline/README.md [hunk 1]
```diff
@@ -53,6 +36,23 @@ privileges you will need to build these tools from source, or bug a sysadmin abo
     1. Toil         pip install toil
     2. S3AM         pip install --pre s3am (optional, needed for uploading output to S3)
 
+## Installation
+
+If Toil is already installed globally (true for CGCloud users), or there are global dependencies (like Mesos),
+use virtualenv's `--system-site-packages` flag and *skip* running `pip install toil`.
+
+Type: `toil-exome` to get basic help menu and instructions
```
Comment (hannes-ucsc) [inline src/toil_scripts/exome_variant_pipeline/README.md]: In my grammar book, there is no colon after Type.

File: src/toil_scripts/exome_variant_pipeline/README.md [hunk 2]
```diff
@@ -53,6 +36,23 @@ privileges you will need to build these tools from source, or bug a sysadmin abo
...
+
+To decrease the chance of versioning conflicts, install toil-scripts into a virtualenv: 
+
+- `virtualenv ~/toil-scripts` 
+- `source ~/toil-scripts/bin/activate`
+- `pip install toil`
+- `pip install toil-scripts`
+
+If Toil is already installed globally (true for CGCloud users), or there are global dependencies (like Mesos),
```
Comment (hannes-ucsc) [inline src/toil_scripts/exome_variant_pipeline/README.md]: Dupe of above. This whole section is ordered awkwardly. I would do it like that:

1) General installation instruction, with a `# CGCLoud users see below` note right next to the `pip install toil` step
2) Exceptions to the general instruction
3) Usage instructions

**Comment by hannes-ucsc** [comment]
John, wait with pushing changes until after the needs work flag goes up. Otherwise you risk inadvertently killing my line notes without having seen them.

File: src/toil_scripts/exome_variant_pipeline/README.md (hunk 2 above)
Comment (jvivian) [inline src/toil_scripts/exome_variant_pipeline/README.md]: There's also a stupid duplication in there as well.

File: src/toil_scripts/bwa_alignment/README.md [hunk 3]
```diff
@@ -44,13 +27,34 @@ privileges you will need to build these tools from source, or bug a sysadmin abo
     1. Toil         pip install toil
     2. S3AM         pip install --pre s3am (optional, needed for uploading output to S3)
 
+## Installation
+
+#### General Installation
```
Comment (hannes-ucsc) [inline src/toil_scripts/bwa_alignment/README.md]: Remove

File: src/toil_scripts/bwa_alignment/README.md [hunk 4]
```diff
@@ -44,13 +27,34 @@ privileges you will need to build these tools from source, or bug a sysadmin abo
...
+or `pip install --pre toil-scripts` for cutting edge development version.
+
+To decrease the chance of versioning conflicts, install toil-scripts into a virtualenv: 
+
+- `virtualenv ~/toil-scripts` 
+- `source ~/toil-scripts/bin/activate`
+- `pip install toil`
+- `pip install toil-scripts`
+
+#### Exceptions
```
Comment (hannes-ucsc) [inline src/toil_scripts/bwa_alignment/README.md]: Remove, these headings are overkill.

File: src/toil_scripts/bwa_alignment/README.md [hunk 5]
```diff
@@ -44,13 +27,34 @@ privileges you will need to build these tools from source, or bug a sysadmin abo
...
+
+If Toil is already installed globally (true for CGCloud users), or there are global dependencies (like Mesos),
```
Comment (hannes-ucsc) [inline src/toil_scripts/bwa_alignment/README.md]: > If there is an existing, system-wide installation of Toil, as is the case when using CGCloud, the `pip install toil` step should be skipped and virtualenv should be invoked with `--system-site-packages`. That way the existing Toil installation will be available inside the virtualenv.

> The reason that Toil isn't installed automatically as a dependency of `toil-scripts` is to give users the opportunity to customize the Toil installation by adding optional extras, e.g. via `pip install toil[aws,mesos]`.

File: src/toil_scripts/bwa_alignment/README.md [hunk 6]
```diff
@@ -44,13 +27,34 @@ privileges you will need to build these tools from source, or bug a sysadmin abo
...
+#### General Installation
+
+Toil-scripts is now pip installable! `pip install toil-scripts` for a toil-stable version 
+or `pip install --pre toil-scripts` for cutting edge development version.
+
+To decrease the chance of versioning conflicts, install toil-scripts into a virtualenv: 
+
+- `virtualenv ~/toil-scripts` 
+- `source ~/toil-scripts/bin/activate`
+- `pip install toil`
```
Comment (hannes-ucsc) [inline src/toil_scripts/bwa_alignment/README.md]: add `but see next paragraph`

**Comment by hannes-ucsc** [comment]
@jvivian done with review.

File: src/toil_scripts/bwa_alignment/README.md (hunk 5 above)
Comment (jvivian) [inline src/toil_scripts/bwa_alignment/README.md]: This is great, thanks!

File: src/toil_scripts/bwa_alignment/README.md [hunk 7]
```diff
@@ -44,13 +27,36 @@ privileges you will need to build these tools from source, or bug a sysadmin abo
...
+
+To decrease the chance of versioning conflicts, install toil-scripts into a virtualenv: 
+
+- `virtualenv ~/toil-scripts` 
+- `source ~/toil-scripts/bin/activate`
+- `pip install toil`, but see next paragraph
+- `pip install toil-scripts`
+
+If there is an existing, system-wide installation of Toil, as is the case when using CGCloud, 
+the pip install toil step should be skipped and virtualenv should be invoked with `--system-site-packages`.
```
Comment (hannes-ucsc) [inline src/toil_scripts/bwa_alignment/README.md]: backquotes around `pip install toil`

File: src/toil_scripts/bwa_alignment/README.md [hunk 8]
```diff
@@ -44,13 +27,36 @@ privileges you will need to build these tools from source, or bug a sysadmin abo
...
+- `pip install toil`, but see next paragraph
+- `pip install toil-scripts`
+
+If there is an existing, system-wide installation of Toil, as is the case when using CGCloud, 
+the pip install toil step should be skipped and virtualenv should be invoked with `--system-site-packages`. 
+That way the existing Toil installation will be available inside the virtualenv.
+
+The reason that Toil isn't installed automatically as a dependency of toil-scripts is to
+ give users the opportunity to customize the Toil installation by adding optional extras, 
+ e.g. via pip install toil[aws,mesos].
```
Comment (hannes-ucsc) [inline src/toil_scripts/bwa_alignment/README.md]: backquotes

**Comment by jvivian** [comment]
@hannes-ucsc — Ready

**Pull request merged**